### DISCUSSION TITLE
Contact Page - Email

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#99] [Feedback]
There is no generic contact@an-nadaa.com

We are giving out his email in all our usual social media profiles and messages.

This email has been left out deleberately or it will be added in a later revision?

**Comment by an-nadaa** [comment]
The form submits to the contact@an-nadaa.com email. do you want it to be mentioned still? we can replace it with the join us one and put general. but it will be redundant (which might not be a bad thing).
